XCGLogger/Library/XCGLogger/XCGLogger.swift

```diff
@@ -203,8 +206,9 @@ public class XCGNSLogDestination: XCGBaseLogDestination {
         else {
             adjustedText = text
         }
-
-        NSLog("%@", adjustedText)
+        dispatch_async(queue) {
+            NSLog("%@", adjustedText)
```

Comment:
Maybe all the work to create the string can also happen in the BG? That way we eliminate all the overhead from (potentially) the main thread :)